static/styles/lib/bootstrap/scss/_input-group.scss

```diff
@@ -136,52 +135,53 @@
 .input-group-btn:not(:first-child) > .dropdown-toggle,
 .input-group-btn:not(:last-child) > .btn:not(:first-child),
 .input-group-btn:not(:last-child) > .btn-group:not(:first-child) > .btn {
-  @include border-left-radius(0);
+	@include border-left-radius(0);
 }
 .form-control + .input-group-addon:not(:first-child) {
-  border-left: 0;
+	border-left: 0;
+	padding: 0.25rem;
```

Comment:
It seems we are using some variables for the paddings in this file. There is also one $input-padding-y-sm which is defined with .25rem. Is it useful to use this variable or create one for this padding or should we leave it like this?

Reply:
you are right, I will change that